app/controllers/uploads_controller.rb

```diff
@@ -57,7 +57,12 @@ def new_upload(csv_type)
     upload = Upload.new(csv_type: csv_type)
     upload.skip_lines = defaults(csv_type)['skip_lines']
     upload.col_sep = defaults(csv_type)['col_sep']
-
+    unless defaults(csv_type)['force_simple_split'].nil?
+      upload.force_simple_split = defaults(csv_type)['force_simple_split']
+    end
+    unless defaults(csv_type)['strip_chars_from_headers'].nil?
+      upload.strip_chars_from_headers = defaults(csv_type)['strip_chars_from_headers']
+    end
```

Comment:
Looks like `force_simple_split` and `strip_chars_from_headers` default to `nil` anyway, so I don't think you need these conditionals.

Reply:
Ripped out the conditionals. Thanks!